.mailmap

```diff
@@ -1,6 +1,7 @@
 Aron Ahmadia <aron@ahmadia.net>
 Phillip Alderman <phillip.d.alderman@gmail.com>
-James Allen <james@sharelatex.com> <jamesallen0108@gmail.com>
+James Allen <james@sharelatex.com>
```

Comment:
Why this change? From https://www.kernel.org/pub/software/scm/git/docs/git-shortlog.html what we want is the line that you remove.

> Proper Name <proper@email.xx> Commit Name <commit@email.xx>
>
> which allows mailmap to replace both the name and the email of a commit matching both the specified commit name and email address.